src/test/regress/expected/aqumv.out

```diff
@@ -3173,31 +3173,23 @@ abort;
 create table par(a int, b int, c int) partition by range(b)
     subpartition by range(c) subpartition template (start (1) end (3) every (1))
     (start(1) end(3) every(1));
-NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'a' as the Cloudberry Database data distribution key for this table.
 HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
```

Comment:
suggest to delete the HINT line as well, it appears together with the above notice line.